src/Sylius/Component/Attribute/Model/Attribute.php

```diff
@@ -170,4 +170,12 @@ public function getStorageType()
     {
         return $this->storageType;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isValueTranslatable()
+    {
+        return AttributeValueInterface::STORAGE_TEXT == $this->getStorageType() ? true : false;
```

Comment:
``return AttributeValueInterface::STORAGE_TEXT === $this->storageType`` would be enough. Moreover, in the future there could be more translatable ``AttributeValue``s (e.g. ``textarea``), so we should deeply consider is it the best choice to check that.